exporting/README.md

```diff
@@ -49,4 +49,269 @@ send charts matching = system.active_processes
+
+        metrics are sent to the service in `JSON` format.
+
+    -   **Google Cloud Pub/Sub Service**
+
+        metrics are sent to the service in `JSON` format.
+
+    -   **MongoDB**
+
+        metrics are sent to the database in `JSON` format.
+
+2.  Netdata can filter metrics (at the chart level), to send only a subset of the collected metrics.
+
+3.  Netdata supports three modes of operation for all exporting connectors:
+
+    -   `as-collected` sends to external databases the metrics as they are collected, in the units they are collected.
+        So, counters are sent as counters and gauges are sent as gauges, much like all data collectors do. For example,
+        to calculate CPU utilization in this format, you need to know how to convert kernel ticks to percentage.
+
+    -   `average` sends to external databases normalized metrics from the Netdata database. In this mode, all metrics
+        are sent as gauges, in the units Netdata uses. This abstracts data collection and simplifies visualization, but
+        you will not be able to copy and paste queries from other sources to convert units. For example, CPU utilization
+        percentage is calculated by Netdata, so Netdata will convert ticks to percentage and send the average percentage
+        to the external database.
+
+    -   `sum` or `volume`: the sum of the interpolated values shown on the Netdata graphs is sent to the external
+        database. So, if Netdata is configured to send data to the database every 10 seconds, the sum of the 10 values
+        shown on the Netdata charts will be used.
+
+    Time-series databases suggest to collect the raw values (`as-collected`). If you plan to invest on building your
+    monitoring around a time-series database and you already know (or you will invest in learning) how to convert units
+    and normalize the metrics in Grafana or other visualization tools, we suggest to use `as-collected`.
+
+    If, on the other hand, you just need long term archiving of Netdata metrics and you plan to mainly work with
+    Netdata, we suggest to use `average`. It decouples visualization from data collection, so it will generally be a lot
+    simpler. Furthermore, if you use `average`, the charts shown in the external service will match exactly what you
+    see in Netdata, which is not necessarily true for the other modes of operation.
+
+4.  This code is smart enough, not to slow down Netdata, independently of the speed of the external database server. You
+    should keep in mind though that many exporting connector instances can consume a lot of CPU resources if they run
+    their batches at the same time. You can set different update intervals for every exporting connector instance, but
+    even in that case they can occasionally synchronize their batches for a moment.
+
+## configuration
+
+In `/etc/netdata/exporting.conf` you should have something like this:
+
+```conf
+[exporting:global]
+enabled = yes
```

Comment:
Please, add a tab for each option that is not a section.

Reply:
Done